infra/launch.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
```

Comment:
ok, actually having tried this out now, it's kinda super slow for first time getting started. Is this just the "base image" slowness?

Do you think we can have a "base base image" that we host on a public docker registry (ghcr or something) that speeds this up a bit more for first time (with a possibly stale install of most deps or something)?

Reply:
Yeah, I was thinking the same thing. It is a one-time cost for your development environment, but yeah, it has to install everything from scratch and then push it to GCP for the first build/launch. I went ahead and split the Dockerfiles into base and incremental: you'll still need to download the base layers to your local machine, but this should be as fast as your internet connection.

For the base image: I'd like to use Dockerhub but I can't see a way to create a "levanter" repo without creating a paid organization. Using GCP is better performance wise: you won't have to "re-push" the base layers you download when building, but there's some egress cost to pay.

I have it pointing at my image for testing, but if you can push to the Stanford repo (or point me at a project to push to) we can use that instead.

Reply:
Yeah Docker is stingy about free orgs now. We can use github's repo for the base image? I'll try that out. (I'll also see about granting you repo access)